### PR TITLE
Update URLs with intended paths

### PR DESCRIPTION
**Pull request by byrnHDF**

### CODE REVIEW

**Comment by lrknox** [comment]
markdown-links-tests found these dead links:
https://support.hdfgroup.org/downloads/hdf5 
https://support.hdfgroup.org/documentation/hdf5 
https://github.com/HDFGroup/hdf5/releases/tag
https://support.hdfgroup.org/documentation/hdf5
https://support.hdfgroup.org/documentation/hdf5/index.html
https://support.hdfgroup.org/downloads/hdf5
Are they dead because they are old links that need correction, or are they links that will be coming soon?  If the PR is merged to develop, presumably these failures will continue in all develop PRs as long as they are dead links.

**Comment by byrnHDF** [comment]
Converted to draft as new links are not working yet

**Comment by bmribler** [comment]
Correct links underneath:

https://support.hdfgroup.org/downloads/hdf5
    -> https://portal.hdfgroup.org/downloads/index.html

https://support.hdfgroup.org/documentation/hdf5
    -> https://portal.hdfgroup.org/documentation/index.html

https://github.com/HDFGroup/hdf5/releases/tag
https://support.hdfgroup.org/documentation/hdf5
    -> same as the second link?

https://support.hdfgroup.org/documentation/hdf5/index.html
    -> should be https://portal.hdfgroup.org/documentation/index.html as well?

https://support.hdfgroup.org/downloads/hdf5
    -> same link as the first link?

**Comment by byrnHDF** [comment]
> Correct links underneath:
> 
> https://support.hdfgroup.org/downloads/hdf5 -> https://portal.hdfgroup.org/downloads/index.html
> 
> https://support.hdfgroup.org/documentation/hdf5 -> https://portal.hdfgroup.org/documentation/index.html
> 
> https://github.com/HDFGroup/hdf5/releases/tag https://support.hdfgroup.org/documentation/hdf5 -> same as the second link?
> 
> https://support.hdfgroup.org/documentation/hdf5/index.html -> should be https://portal.hdfgroup.org/documentation/index.html as well?
> 
> https://support.hdfgroup.org/downloads/hdf5 -> same link as the first link?

No this PR is for new links going forward.

File: doxygen/aliases [hunk 1]
```diff
@@ -3,18 +3,25 @@ ALIASES += THG="The HDF Group"
...
-ALIASES += ARCURL="docs.hdfgroup.org/archive/support/HDF5/doc"
-ALIASES += RFCURL="docs.hdfgroup.org/hdf5/rfc"
-ALIASES += DSPURL="portal.hdfgroup.org/display/HDF5"
-ALIASES += DOCURL="portal.hdfgroup.org/documentation/hdf5-docs"
-ALIASES += DWNURL="portal.hdfgroup.org/downloads"
-ALIASES += AEXURL="support.hdfgroup.org/ftp/HDF5/examples"
+# Default URL for HDF Group Files
+ALIASES += HDFURL="support.hdfgroup.org"
+# URL for archived files
+ALIASES += ARCURL="\HDFURL/archive/support/ftp/HDF5"
```
Comment (byrnHDF) [inline doxygen/aliases:9]: This needs to be reviewed and maybe changed to something that makes more sense.

File: doxygen/aliases [hunk 2]
```diff
@@ -3,18 +3,25 @@ ALIASES += THG="The HDF Group"
...
+# URL for RFCs
+ALIASES += RFCURL="\HDFURL/documentation/hdf5/rfc"
+# URL for documentation
+ALIASES += DOCURL="\HDFURL/documentation/hdf5"
+# URL for downloads
+ALIASES += DWNURL="\HDFURL/downloads/hdf5"
+ALIASES += AEXURL="\HDFURL/ftp/HDF5/examples"
```
Comment (byrnHDF) [inline doxygen/aliases:16]: This can't just point to the source HDF5Examples folder, because of Tutorial files.

File: doxygen/aliases (hunk 1 above)
Comment (bmribler) [reply doxygen/aliases:9]: I know that "archive" shouldn't be used.

**Comment by byrnHDF** [comment]
Replaced by #4549

**Pull request closed without merging**